rootfs/api/models/app.py

```diff
@@ -23,6 +23,7 @@
 from api.models.release import Release
 from api.models.config import Config
 from api.models.domain import Domain
+from api.models.appsettings import AppSettings
```

Comment:
I almost feel as if this should be part of another package dir...

```
from api.models.app.settings import Settings
```

Any thoughts on the reasoning behind this package naming scheme? I mean it's directly tied to the app model...